mesonbuild/interpreter.py

```diff
@@ -2022,6 +2022,8 @@ def func_build_target(self, node, args, kwargs):
             raise InterpreterException('Unknown target_type.')
 
     def func_vcs_tag(self, node, args, kwargs):
+        if 'input' not in kwargs or not 'output' in kwargs:
```

Comment:
__[flake8]__
* [E713] test for membership should be 'not in'

<!-- Comment by [SideCI](https://sideci.com) -->